From 01e947f861614e0ff2fdc1f44c77f4da7813f6eb Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Wed, 20 May 2015 15:49:38 +0800
Subject: [PATCH] MLK-10969-1 ARM: dts: imx7d: fill mipi_dsi node with
 required properties

Fill the 'interrupts', 'clocks' and 'mipi-phy-supply'
properties to the mipi_dsi node which are necessary
for the mipi_dsi driver.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 959c0bc..39bb5af 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -835,6 +835,11 @@
 			mipi_dsi: mipi-dsi@30760000 {
 				compatible = "fsl,imx7d-mipi-dsi";
 				reg = <0x30760000 0x10000>;
+				interrupts = <GIC_SPI 41 IRQ_TYPE_LEVEL_HIGH>;
+				clocks = <&clks IMX7D_MIPI_DSI_ROOT_CLK>,
+					<&clks IMX7D_MIPI_DPHY_ROOT_CLK>;
+				clock-names = "mipi_cfg_clk", "mipi_pllref_clk";
+				mipi-phy-supply = <&reg_1p0d>;
 				status = "disabled";
 			};
 
-- 
1.8.0

